Builds/VisualStudio2015/RippleD.vcxproj

```diff
@@ -3353,6 +3353,10 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug|x64'">True</ExcludedFromBuild>
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release|x64'">True</ExcludedFromBuild>
     </ClCompile>
+    <ClCompile Include="..\..\src\ripple\rpc\tests\AccountOffers.test.cpp">

```

Comment:
so I edited these project files by hand and did so by copying lines above, etc. In my editor I'm seeing 0x0D 0x0A for every line in this file, but maybe I'm misreading. If the file can't be opened by visual studio, I can investigate more.